plugin/jvm/src/main/java/com/twosigma/beaker/chart/serializer/ObservableChartSerializer.java

```diff
@@ -0,0 +1,27 @@
+package com.twosigma.beaker.chart.serializer;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.google.inject.Singleton;
+import com.twosigma.beaker.chart.ObservableChart;
+import com.twosigma.beaker.chart.actions.ChartObjectManager;
+import com.twosigma.beaker.jvm.updater.UpdateManager;
+import org.codehaus.jackson.JsonGenerator;
+import org.codehaus.jackson.map.JsonSerializer;
+
+import java.io.IOException;
+
+@Singleton
+public abstract class ObservableChartSerializer<T extends ObservableChart> extends JsonSerializer<T> {
```

Comment:
@mnavasiolava Why is this both a Singleton and abstract? I don't know much about Guice so I might be missing something but I don't see how can an abstract class be a Singleton, or be instantiated at all.